docs/johannes_mueller/yolo_from_omero/train_yolo.ipynb

```diff
@@ -0,0 +1,705 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Object class: 0, Box: [     65.003      70.267      9.9182      11.387]\n",
+      "Object class: 0, Box: [     89.628      61.473        12.1      12.812]\n",
+      "Object class: 0, Box: [     62.141      88.931       9.556      10.735]\n",
+      "Object class: 0, Box: [     80.845      69.067       9.647      11.756]\n",
+      "Object class: 0, Box: [     81.321      59.599      10.501      11.759]\n",
+      "Object class: 0, Box: [     73.621      82.333      11.683      10.676]\n",
+      "Object class: 0, Box: [     61.595      77.768      8.4934      9.3834]\n"
+     ]
+    }
+   ],
+   "source": [
+    "for obj_class, box in zip(prediction[0].boxes.cls, prediction[0].boxes.xywh):\n",
+    "    print(f'Object class: {int(obj_class)}, Box: {box.detach().cpu().numpy()}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Moving the data back to OMERO\n",
+    "\n",
+    "To come full circle, here's a quick glance on how to move the images along with the predictions back to OMERO. Luckily for us, ezomero makes this relatively straightforward:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "swapped_object_classes = {value: key for key, value in object_classes.items()}\n",
+    "\n",
+    "rectangles = []\n",
+    "for obj_class, box in zip(prediction[0].boxes.cls, prediction[0].boxes.xywh):\n",
+    "    box_numpy = box.detach().cpu().numpy()\n",
+    "    x = box_numpy[0] - box_numpy[2] / 2\n",
+    "    y = box_numpy[1] - box_numpy[3] / 2\n",
+    "    width = box_numpy[2]\n",
+    "    heigth = box_numpy[3]\n",
+    "    \n",
+    "    rectangles.append(ezomero.rois.Rectangle(x, y, width, heigth, label=swapped_object_classes[int(obj_class)]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lastly, we could upload this image and the corresponding predictions to a new dataset in OMERO. Note that the image has to be converted in 5d (in our case, we set the format to `tczyx`) to be compatible with OMERO.\n",
+    "\n",
+    "```{note}\n",
+    " The steps up to this point can take a bit of time so you may have to reconnect to the omero server.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_to_upload = io.imread(test_image)\n",
+    "image_id = ezomero.post_image(\n",
+    "    conn, image=image_to_upload[None, None, None, :], dim_order='tczyx',\n",
+    "    dataset_id=260, image_name='example_prediction'\n",
```

Comment:
How would I know which `dataset_id ` to use here?

I saw that in the download step the dataset id was 259. So 260 would obviously make sense. But what if I am not the only user on the Omero server and 260 has already been taken by somebody else?